src/patternfly/components/Button/button-icon.hbs

```diff
@@ -1,5 +1,6 @@
-<i class="fas{{#if button-icon--type}} fa-{{button-icon--type}}{{/if}}{{#if button-icon--modifier}} {{button-icon--modifier}}{{/if}}"
-  aria-hidden="true"
+<i class="fas{{#if button-icon--type}} fa-{{button-icon--type}}{{/if}}{{#if button-icon--modifier}} {{button-icon--modifier}}{{/if}}" 
+aria-hidden="true"
   {{#if button-icon--attribute}}
     {{{button-icon--attribute}}}
-  {{/if}}></i>
\ No newline at end of file
+  {{/if}}>
+</i>
```

Comment:
Can you revert the change for this file so that the PR shows no changes to this file?

Reply:
This file has been reverted back as well.